examples/basic.zig

```diff
@@ -25,4 +25,5 @@ test "bench test basic" {
     };
     defer benchmarkResults.results.deinit();
     try zbench.run(myBenchmark, &bench, &benchmarkResults);
+    try benchmarkResults.prettyPrint();
```

Comment:
What kind of benefit do we have here that this is an extra function on the benchmarkResults? For me this feels kind of unnecessary. From a user perspective i just want to run an benchmark, with the least amount on code.

Reply:
1. When we want to print multiple results under the same header, we first complete all the benchmarks using
     `zbench.run` then once all the results are collected we do `.prettyPrint`. If `zbench.run` displays each result
     individually then each result gets printed under a new header which is not always what we want.
2. Later on we might get other parsing options other than `prettyPrint`.

Reply:
Ah true, good use case. Haven't thought of this 👍